pkg/restic/restic.go

```diff
@@ -471,7 +471,8 @@ func SnapshotIDsFromSnapshotCommand(output string) ([]string, error) {
 // SpaceFreedFromPruneLog gets the space freed from the prune log output
 // For reference, here is the logging command from restic codebase:
 // Verbosef("will delete %d packs and rewrite %d packs, this frees %s\n",
-//		len(removePacks), len(rewritePacks), formatBytes(uint64(removeBytes)))
+//
+//	len(removePacks), len(rewritePacks), formatBytes(uint64(removeBytes)))
```

Comment:
tiny nit: this looks like it's supposed to be formatted:

```suggestion
//
//	Verbosef("will delete %d packs and rewrite %d packs, this frees %s\n",
//		len(removePacks), len(rewritePacks), formatBytes(uint64(removeBytes)))
```